test.log

```diff
@@ -0,0 +1,2359 @@
+  >  Running go vet
```

Comment:
I'm not sure if this file should be included in this PR. As far as this file only contains logs, I suggest remove it